tests/Unit/Evolution/Systems/CurvedScalarWave/Test_UpwindFlux.cpp

```diff
@@ -184,7 +186,7 @@ void test_upwind_flux_random() noexcept {
   // (except that the normal vector gets multiplied by -1.0) that the
   // numerical flux reduces to the flux
   INFO("test consistency of the curved-scalar-wave upwind flux")
-  CurvedScalarWave::UpwindFlux<Dim> flux_computer{};
+  CurvedScalarWave::UpwindPenaltyCorrection<Dim> flux_computer{};
```

Comment:
Please rename this file `Test_UpwindFlux.cpp` -> `Test_UpwindPenaltyCorrection.cpp`

Reply:
done